changelog.d/6656.bugfix

```diff
@@ -0,0 +1 @@
+No more overriding the entire /etc folder of the container in docker-compose.yaml. Contributed by Fabian Meyer.
```

Comment:
since this isn't fixing a bug in the core code and is more of a documentation fix: could you rename this file to `6656.doc` ?